huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go

```diff
@@ -656,6 +799,25 @@ func resourceFgsFunctionRead(_ context.Context, d *schema.ResourceData, meta int
 		setFuncionMountConfig(d, f.MountConfig),
 		d.Set("versions", versionConfig),
 	)
+
+	reservedInstances, err := getReservedInstanceConfig(fgsClient, d)
+	if err != nil {
+		mErr = multierror.Append(mErr, err)
+	} else if len(reservedInstances) > 0 {
+		reservedInstance := reservedInstances[0]
+		qualifierName := reservedInstance.QualifierName
+		if (reservedInstance.QualifierType) == "alias" && reservedInstance.QualifierName != "" {
```

Comment:
Why not configure an Optional type field of qualifier_type？

Reply:
fixed, add qualifier_type parameter